### ISSUE TITLE
What's with the big hole below the tower of hanoi?

### ISSUE THREAD

**Issue by gitcnd** [kickstarter/kickstarter-autodesk-3d#25]
The underside is not flat on the bed - is that supposed to be testing something, or is it just to annoy us ?  (We can't add supports because then the overhang test won't work, ...)

**Comment by gitcnd** [comment]
It is "Extrude 31" (a cut) in the timeline.

**Comment by davispw** [comment]
@gitcnd What's annoying about it?  It's not part of the test protocol but it does a great job of showing bottom skin quality for bridging.  No need for supports.

**Comment by gitcnd** [comment]
because the entire print will fail if this "bridge for no reason" fails - remember - this is designed to be printed on machines that are not yet properly set up.

**Comment by GitWyd** [comment]
@gitcnd If a printer fails on basic overhangs, I would argue this is not the right test print. The KSR benchmark is more for performance evaluation of a well working printer, rather than initial tuning of a broken or poorly calibrated machine. Maybe start with testing bridging, stringing, overhangs, and dimensional accuracy first using separate prints and get your machine dialed in, then come back to KSR and see how well the machine is doing in comparison to other machines.

**Comment by gitcnd** [comment]
@GitWyd - you'd lose that argument for sure.  This is a TEST PRINT that is specifically designed to be used to help you set up a printer !!

Having bridges under the part which is designed for testing dimensional accuracy makes no sense - the tests for bridging/overhangs/etc are *different* parts of this model already.

**Comment by GitWyd** [comment]
@gitcnd Is it though? From what I gather, it is "a common standard that enables people to assess the performance of FDM 3D printers." Maybe I am wrong, but there seems to be an expectation of basic functionality e.g. a performance baseline

I do agree with you though that this hollow space should be removed from the design unless it is used for scoring in the Assessment Protocol.

**Comment by gitcnd** [comment]
@GitWyd yes, they clearly state that the purpose of the test ..."provides something discrete to measure and compare as relevant variables are changed." - like I said: this is specifically designed to be used to help you set up a printer - (that's what you do when setting up a printer - change the variables to get the test print to work the best).
